### ISSUE TITLE
Assertion failed: (!ppe_pending), function ngtcp2_conn_writev_stream, file ngtcp2_conn.c, line 8210.

### ISSUE THREAD

**Issue by amirlivneh** [ngtcp2/ngtcp2#234]
See log at https://gist.github.com/amirlivneh/c8e80517b93da62dc33fa469a77951bf

This happens sporadically but can be quickly reproduced by repeatedly uploading a 1 MB file from the example client to the example server:
```
mkfile 1M /tmp/1M
./ci/gen-certificate.sh
./examples/server 127.0.0.1 9999 cert/server.key cert/server.crt
while ./examples/client 127.0.0.1 9999 https://127.0.0.1/test --exit-on-first-stream-close -d /tmp/1M -m POST && echo "------------\n"; do :; done
```

**Comment by tatsuhiro-t** [comment]
Thank you for telling me this bug.  I think it was fixed via commit bed13a9

**Comment by amirlivneh** [comment]
It appears to be fixed now, thanks!
